### PR TITLE
feat: add footer design and first section

### PR DESCRIPTION
**Pull request by mkubdev**
## Related Issue 

- Information about the related issue
N/A

Closes: # 

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

 Original           | Updated
 :--------------------: |:--------------------:
133555103-20b3e505-a9af-4829-aa77-dd6b089f2fe0 (1) | Capture |

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,7 +36,7 @@
         <div class="pl-4 flex items-center">
           <a class="toggleColour text-black no-underline hover:no-underline font-bold text-2xl lg:text-2xl" href="#">
             <!--Icon from: http://www.potlabicons.com/ -->
-            Web3 Community
+            <img src="./logo.png" alt="Web3 Community" class="h-16 items-start">
```
Comment (vinzvinci) [inline index.html:39]: Add space at line 39 "Web 3 Community"

Comment (shuklaritvik06) [reply index.html]: Yess

**Review by shuklaritvik06 — COMMENTED** [review]
Everything else looks good just add a space as vinz said :)

**Review by vinzvinci — APPROVED** [review]
LGTM 👍 We can keep improving the others, I think you can merge this.

**Review by shuklaritvik06 — APPROVED** [review]
LGTM :) You can merge :)

File: index.html (hunk 1 above)
Comment (mkubdev) [reply index.html]: Fixed it with https://github.com/mkubdev/web3community.github.io/commit/5b64ecfc3d2f406b4d6ba88e6813c681ba620334

**Pull request merged**